applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -31,7 +33,7 @@ CoulombYieldSurface::KappaDependentFunction MakeConstantFunction(double Value)
 
 CoulombYieldSurface::KappaDependentFunction MakeLinearFunction(double Value, double Coefficient)
 {
-    return [Value, Coefficient](double kappa) { return std::max(Value + Coefficient * kappa, 0.0); };
+    return [Value, Coefficient](double kappa) { return Value + Coefficient * kappa; };
```

Comment:
Nitpicking (to follow the Kratos Style Guide):
```suggestion
    return [Value, Coefficient](double Kappa) { return Value + Coefficient * Kappa; };
```